amazon-sqs/5.11/modules/ROOT/pages/index.adoc

```diff
@@ -9,6 +9,10 @@ The AWS SDK for Java provides a Java API for AWS infrastructure services. Amazon
 
 For compatibility information and fixed issues, see the Amazon SQS Connector Release Notes.
 
+== Grant Amazon SQS API Permissions to Perform Actions
+
+To use Amazon SQS Connector, you must first ensure to configure your AWS user to operate an AWS resource. For the configuration, you must grant API permissions to perform Amazon SQS actions, that enables you to set up access control and write permissions policies that you can attach to an IAM identity. Refer to xref:https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-api-permissions-reference.html[Amazon SQS API permissions] documentation for details.
```

Comment:
In the second sentence, change the first "that" to "which"

"For the configuration, you must grant API permissions to perform Amazon SQS actions, which enables you to..."